setup.md

```diff
@@ -20,8 +20,11 @@ Once complete, click **Next**.
 ![Account Recovery Key](/assets/images/signing/recovery_key.png)
 
 Next, you will presented with your account recovery key.
-Please save this to a secure location, this is the ONLY time you will be shown.
-We do not store your own recovery key, if you lose this we will be unable to recover your data attached to the account.
+
+**_Please save your account recovery key to a secure location, this is the ONLY time it will be shown._**
+
+**_We do not store your own recovery key, if you lose this we will be unable to recover your data attached to the account._**
```

Comment:
Lets tell the user "why?"? I.e. this is because we use end to end encryption to ensure the highest security of your data.